config/environments/test.rb

```diff
@@ -57,8 +54,15 @@
   config.x.default_host = 'lvh.me'
   config.x.client_port = 3200
   config.x.server_port = 7979
+  config.x.default_app_host = "#{config.x.default_host}:3200"
```

Comment:
`config.x.default_app_host = "#{config.x.default_host}:#{config.x.client_port}"`
minimize use of magic numbers.
Your current code means that if you change your client port from 3200 to something else, you need to change two places, which may lead to oversight and bugs later on.
https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad

Reply:
noted on this one..
also worth noting that previously in the `development` environment, the `client_port` over there was set to `5000`, which was the previous value used for this one. I will refactor this for more clarity